src/core/Akka.Remote/Transport/DotNetty/DotNettyTransportSettings.cs

```diff
@@ -269,18 +269,21 @@ private static SslSettings Create(Config config)
             if (config.IsNullOrEmpty())
                 throw new ConfigurationException($"Failed to create {typeof(DotNettyTransportSettings)}: DotNetty SSL HOCON config was not found (default path: `akka.remote.dot-netty.tcp.ssl`)");
 
+            var requireMutualAuth = config.GetBoolean("require-mutual-authentication", true);
```

Comment:
new setting

Reply:
Ah, we need to update the Akka.Remote `ConfgSpec` to make sure this is set to `true` by default.

We probably also need another spec to validate that mutual TLS can be disabled via this setting as well (like an integration test, not a config test)

Reply:
Done